spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java

```diff
@@ -247,6 +255,7 @@ private void deployStream(StreamDefinition stream, Map<String, String> streamDep
 			isDownStreamAppPartitioned = isPartitionedConsumer(appDeployTimeProperties,
 					upstreamAppSupportsPartition);
 
+			logger.info(String.format("Downloading resource URI [%s]", registration.getUri()));
 			Resource appResource = registration.getResource();
 			Resource metadataResource = registration.getMetadataResource();
```

Comment:
ok, so I guess just let's leave the logging for the big jar file.  That is what is really impacting the experience.